x501/src/time.rs

```diff
@@ -51,6 +53,15 @@ impl Time {
     }
 }
 
+impl fmt::Display for Time {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> core::result::Result<(), fmt::Error> {
+        match self {
+            Time::UtcTime(t) => f.write_str(t.to_date_time().to_string().as_str()),
+            Time::GeneralTime(t) => f.write_str(t.to_date_time().to_string().as_str()),
+        }
```

Comment:
Alternatively, you could define a `fn to_date_time()` and then use:

```suggestion
        write!(f, "{}", self.to_date_time())
```

Reply:
Done. Thanks.